src/main/java/com/example/playdocker/models/Score.java

```diff
@@ -0,0 +1,25 @@
+package com.example.playdocker.models;
+
+public class Score {
+    private String threatType;//Typ zagrożenia badany przez API google
+    private String confidenceLevel;//Poziom zagrożenia stwarzany przez badany URL
+
+    public Score() {
+    }
+
+    public String getThreatType() {
```

Comment:
Czy potrzebujesz tego gettera? Czy warto pisać metody, które nie są potrzebne w danym momencie na wypadek gdyby niedługo były? Znasz może zasadę, która to potwierdzi lub temu zaprzeczy? :)

Reply:
Nie potrzebuje, powstał tylko dlatego, że automatycznie wygenerowałem gettery i settery, a potem zapomniałem posprzątać.
Istnieje taki akronim YAGNI, mówiący, że nie będę tego potrzebował, więc lepiej nie pisać nadmiarowego kodu. Pogarsza to czytelność kodu, który jest potrzebny.